docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Garbage collector config settings
+description: Learn about run-time settings for configuring how the garbage collector manages memory.
+ms.date: 11/13/2019
+ms.topic: reference
+---
+# Run-time configuration options for garbage collection
+
+This page contains information about garbage collector (GC) settings that can be changed at run time. If you're trying to achieve peak performance of a running app, consider using these settings. However, most users will not need to concern themselves with these settings.
+
+On this page, settings are arranged into groups of settings that you can use in conjunction with each other to achieve a specific result.
+
+> [!NOTE]
+>
+> - These settings can also be changed dynamically by the app as it's running, so any run-time settings you set may be overridden.
+> - Some settings, such as [latency level](../../standard/garbage-collection/latency.md), are typically set only through the API at design time. Such settings are omitted from this page.
+
+## Flavors of garbage collection
+
+The two main flavors of garbage collection are workstation GC and server GC. For more information about differences between the two, see [Fundamentals of garbage collection](../../standard/garbage-collection/fundamentals.md#workstation-and-server-garbage-collection).
+
+The subflavors of garbage collection are background and non-concurrent.
+
+Use the settings described in this section to select flavors of garbage collection.
+
+### System.GC.Server/COMPlus_gcServer
+
+- Configures whether the application uses workstation garbage collection or server garbage collection.
+- Default: Workstation garbage collection (`false`).
+- For more information, see [Configure garbage collection](../../standard/garbage-collection/fundamentals.md#configuring-garbage-collection).
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.Server` | `true` - server<br/>`false` - workstation | .NET Core 1.0 |
```

Comment:
nit: I think it would be good if the entries would always follow the same order. e.g.

```suggestion
| **runtimeconfig.json** | `System.GC.Server` | `false` - workstation<br/>`true` - server| .NET Core 1.0 |
```

Reply:
I'm going with Bill's suggestion of always putting the default first.